### PR TITLE
.py corrected

### PR DESCRIPTION
**Pull request by LucaBestagno**

### CODE REVIEW

**Comment by dgarijo** [comment]
Remove the .DS_store file please!

File: Assignment4/luca-bestagno-CA37228OQ/task07.py [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+
+# TO DO
+# Visualize the results
+
+from rdflib.plugins.sparql import prepareQuery
+
+q1 = prepareQuery('''
+  SELECT ?subclass
+  WHERE {
+    ?subclass rdfs:subClassOf ns:LivingThing.
```
Comment (dgarijo) [inline Assignment4/luca-bestagno-CA37228OQ/task07.py:34]: this is not retrieving all subclasses of subclasses of living thing. Use `*`

File: Assignment4/luca-bestagno-CA37228OQ/task07.py [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+
+# TO DO
+# Visualize the results
+
+q2 = prepareQuery('''
+  SELECT ?individual
+  WHERE {
+    { ?individual rdf:type ns:Person. }
+    UNION
+    { ?individual rdf:type ?subclass.
```
Comment (dgarijo) [inline Assignment4/luca-bestagno-CA37228OQ/task07.py:55]: same issue as above

File: Assignment4/luca-bestagno-CA37228OQ/task07.py [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+"""First let's read the RDF file"""
+
+from rdflib import Graph, Namespace, Literal
+from rdflib.namespace import RDF, RDFS
+g = Graph()
+g.namespace_manager.bind('ns', Namespace("http://somewhere#"), override=False)
+g.namespace_manager.bind('vcard', Namespace("http://www.w3.org/2001/vcard-rdf/3.0#"), override=False)
+g.parse(github_storage+"/rdf/example6.rdf", format="xml")
+
+"""**TASK 7.1: List all subclasses of "LivingThing" with RDFLib and SPARQL**"""
```
Comment (dgarijo) [inline Assignment4/luca-bestagno-CA37228OQ/task07.py:24]: missing all exercises with rdflib

**Review by dgarijo — CHANGES_REQUESTED** [review]
Please add remaining exercises

**Comment by SerPablo** [comment]
Delivered with some error, no more PRs allowed. Rdflib and small errors.

**Pull request merged**